docs/user_tracking.rst

```diff
@@ -75,8 +83,65 @@ This is very helpful when using ``register``:
     register(Poll, get_user=get_poll_user)
 
 
+.. _`Manually Track User Model`:
+
+
+Manually Track User Model
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Although ``django-simple-history`` tracks the ``history_user`` (the user who changed the
+model) using a django foreign key, there are instances where we might want to track this
+user but cannot use a Django foreign key.
+
+**Note:** If you want to track a custom user model that is still accessible through a
+Django foreign key, refer to `Change User Model`_.
+
+The two most common cases where this feature will be helpful are:
+
+1. You are working on a Django app with multiple databases, and your history table
+   is in a separate database from the user table.
+
+2. The user model that you want to use for ``history_user`` does not live within the
+   Django app, but is only accessible elsewhere (i.e. through an API call).
+
+There are three parameters to ``HistoricalRecords`` or ``register`` that facilitate
+the ability to manually track a ``history_user``.
+
+
+:history_user_id_field: An instance of field (i.e. ``IntegerField(null=True)`` or
```

Comment:
Can you use other types? Should be `e.g.` then?

Reply:
Can be whatever type you want it to be. Any unique field on the user object